config/config.exs

```diff
@@ -33,6 +33,8 @@ config :skate,
   redirect_http?: false,
   static_href: {SkateWeb.Router.Helpers, :static_path},
   timezone: "America/New_York",
+  swings_beta_usernames: {:system, "SWINGS_BETA_USERNAMES"},
+  beta_username_prefix: {:system, "BETA_USERNAME_PREFIX"},
```

Comment:
what's with this prefix? also, you refer to it as a "suffix" in the commits message

Reply:
So for some reason, on the dev user pool the usernames are of the form `ActiveDirectory_jsmith` whereas on prod they're of the form `mbta-active-directory_jsmith`. And yes, if I used "suffix" in the commit message that was in error.